subsys/secure_storage/Kconfig.its_store

```diff
@@ -12,7 +12,7 @@ config SECURE_STORAGE_ITS_STORE_IMPLEMENTATION_SETTINGS
 		&& $(dt_node_has_compat,$(dt_node_parent,$(DT_STORAGE_PARTITION)),fixed-partitions)
 	imply FLASH_MAP
 	imply NVS
```

Comment:
I'm wondering if it would make sense to remove this line as we are changing the paradigm from selecting to depending on...